### DISCUSSION TITLE
the program turns off

### DISCUSSION THREAD

**Discussion by Dmitry081** [MohamedKVIP/MK-PUBG-Mobile-Tool#12] [General]
I have a problem, when I click on the connect Gameloop button, the program turns off and nothing happens, what should I do?

**Comment by MohamedKVIP** [comment]
Hi @Dmitry081 ,

Thanks for reporting this issue. It sounds like you might be encountering a bug that we're addressing in the next update.

In the meantime, you could try our AIO version, which has a different connection method. You can find it on our Discord server: [Link to Discord](https://discord.gg/PDPJM6e6PC).

We'll also have a fix for this specific Gameloop connection problem in the upcoming release. Please stay tuned!
